pandas/src/datetime_helper.h

```diff
@@ -2,16 +2,19 @@
 #include "numpy/arrayobject.h"
 #include "numpy/arrayscalars.h"
 #include <stdio.h>
+#include "helper.h"
 
 #if PY_MAJOR_VERSION >= 3
 #define PyInt_AS_LONG PyLong_AsLong
 #endif
 
+PANDAS_INLINE
 void mangle_nat(PyObject *val) {
   PyDateTime_GET_MONTH(val) = -1;
   PyDateTime_GET_DAY(val) = -1;
 }
 
+PANDAS_INLINE
 npy_int64 get_long_attr(PyObject *o, const char *attr) {
   npy_int64 long_val;
```

Comment:
It is used in https://github.com/pandas-dev/pandas/blob/2e276fb2fec6dd04d6abcf5e79c03853ee86cd24/pandas/src/ujson/python/objToJSON.c#L2129